package-lock.json

```diff
@@ -45,6 +46,11 @@
       "resolved": "https://registry.npmjs.org/@exodus/schemasafe/-/schemasafe-1.3.0.tgz",
       "integrity": "sha512-5Aap/GaRupgNx/feGBwLLTVv8OQFfv3pq2lPRzPg9R+IOBnDgghTGW7l7EuVXOvg5cc/xSAlRW8rBrjIC3Nvqw=="
     },
+    "node_modules/@faker-js/faker": {
```

Comment:
just confirming - all the new imports are necessary, right?

Reply:
Yea I ran a `npm ci` before commit 🙂